components/eventsrv-protocol/build.rs

```diff
@@ -26,7 +26,6 @@ fn generate_protocols() {
     let cmd = Command::new(format!("{}/bin/protoc", prefix))
         .arg("--rust_out")
         .arg(out_dir)
-        .arg("--doc_out=markdown,index.md:doc")
```

Comment:
This generates documentation for the protocols - I don't think I moved the needle enough on automation which is why you were getting the error. It's fine to remove for now!